demos/python_demos/asl_recognition_demo/README.md

```diff
@@ -0,0 +1,85 @@
+# ASL Recognition Python* Demo
+
+This demo demonstrates how to run ASL (American Sign Language) Recognition models using OpenVINO&trade;.
+
+## How It Works
+
+The demo application expects an ASL recognition model in the Intermediate Representation (IR) format.
+
+As input, the demo application takes:
+* a path to a video file or a device node of a web-camera specified with a command line argument `--input`
+* a path to a file in `JSON` format with ASL class names `--class_map`
+
+The demo workflow is the following:
+
+1. The demo application reads video frames one by one, runs person detector that extracts ROI, tracks the ROI of very first person. Additional process is used to prepare the bach of frames with constant framerate.
+2. Batch of frames and extracted ROI are passed to artificial neural network that predicts the ASL gesture.
+3. The app visualizes results of it work as graphical window where following objects are shown:
+    - Input frame with detected ROI.
+    - Last recognized ASL gesture.
+    - Performance characteristics.
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html).
+
+## Running
+
+Run the application with the `-h` option to see the following usage message:
+
+```
+usage: asl_recognition_demo.py [-h] -am ACTION_MODEL -dm DETECTION_MODEL -i
+                               INPUT -c CLASS_MAP [-d DEVICE]
+                               [-l CPU_EXTENSION] [--no_show]
+
+Options:
+  -h, --help            Show this help message and exit.
+  -am ACTION_MODEL, --action_model ACTION_MODEL
```

Comment:
Please, rename this keys as in [contribution guide](https://github.com/opencv/open_model_zoo/blob/develop/CONTRIBUTING.md#demos). (`-m_*`)

Reply:
Fixed